client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-dialog/assignment-poll-dialog.component.ts

```diff
@@ -25,6 +26,9 @@ import { AssignmentPollService, UnknownUserLabel } from '../../services/assignme
 export class AssignmentPollDialogComponent extends BasePollDialogComponent {
     public unknownUserLabel = UnknownUserLabel;
 
+    @ViewChild(AssignmentPollFormComponent, { static: true })
```

Comment:
Provide the `BasePollFormComponent` in the `AssignmentPollFormComponent` and search the selector `BasePollFormComponent` in the `BasePollDialogComponent`